src/service/viewport-impl.js

```diff
@@ -652,8 +670,11 @@ export class ViewportBindingDef {
   /**
    * Updates binding with the new padding.
    * @param {number} unusedPaddingTop
+   * @param {boolean|undefined} unusedOptUpdateScrollPos
+   * @param {number}undefined} unusedLastPaddingTop
```

Comment:
And `unusedOptLastPaddingTop`. After first "opt", all other args must also be "opt".